docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -0,0 +1,78 @@
+---
+approvers:
+- derekwaynecarr
+title: Scheduling HugePages
+---
+
+{% capture overview %}
+
+Kubernetes includes **alpha** support for managing huge pages spread across
```

Comment:
nit: "Kubernetes supports the allocation and consumption of huge pages by applications in a pod as an **alpha** feature."